azurerm/internal/services/web/app_service.go

```diff
@@ -1735,7 +1745,13 @@ func flattenAppServiceIpRestriction(input *[]web.IPSecurityRestriction) []interf
 			if *ip == "Any" {
 				continue
 			} else {
-				restriction["ip_address"] = *ip
+				if v.Tag == web.Default {
+					restriction["ip_address"] = *ip
+				} else if v.Tag == web.ServiceTag {
+					restriction["service_tag"] = *ip
+				} else {
+					// FIXME: I want to error here if we got XffProxy. I don't remember being able to set this in the portal
```

Comment:
The flatten method here doesn't currently allow an error to be returned, should I refactor to allow it or handle this case silently?